doc/source/serve/doc_code/configure_serve_deployment/model_deployment.py

```diff
@@ -1,72 +1,36 @@
 # flake8: noqa
 
+import ray
+
 # __deployment_start__
 # File name: configure_serve.py
 
-import ray
 from ray import serve
-from fastapi import FastAPI
-
-from transformers import pipeline
-
-app = FastAPI()
 
 
 @serve.deployment(
     name="Translator",
     num_replicas=2,
     ray_actor_options={"num_cpus": 0.2, "num_gpus": 0},
     max_concurrent_queries=100,
-    # autoscaling_config={
-    #     "min_replicas": 1,
-    #     "initial_replicas": 2,
-    #     "max_replicas": 5,
-    #     "target_num_ongoing_requests_per_replica": 10
-    # },
-    # user_config={},
     health_check_period_s=10,
     health_check_timeout_s=30,
     graceful_shutdown_timeout_s=20,
     graceful_shutdown_wait_loop_s=2,
 )
-@serve.ingress(app)
-class Translator:
-    def __init__(self):
-        # Load model
-        self.model = pipeline("translation_en_to_fr", model="t5-small")
-
-    @app.post("/")
-    def translate(self, text: str) -> str:
-        # Run inference
-        model_output = self.model(text)
+class Example:
+    ...
 
-        # Post-process output to return only the translation text
-        translation = model_output[0]["translation_text"]
 
-        return translation
-
-
-translator_app = Translator.bind()
+translator_app = Example.bind()
```

Comment:
Can we change this variable to `example_app` in the code and the explanation? There's nothing that indicates that this is a translator app anymore.